thirdparties/brpc/fix-gcc11.patch

```diff
@@ -0,0 +1,463 @@
+diff --git a/BUILD b/BUILD
```

Comment:
Is this patch directly from brpc? If so, it's better to add some related information in commit message, so that we can track it.

Reply:
I generate this patch after apply brpc.patch.

Reply:
I will add more information to first commit point.